Functions.Templates/Templates/DurableFunctionsEntity-JavaScript/metadata.json

```
@@ -0,0 +1,18 @@
+﻿{
+  "defaultFunctionName": "DurableFunctionsEntityJS",
```

```suggestion
  "defaultFunctionName": "Counter",
```

Would it make more sense to default to `Counter`?

---

Not sure if this was resolved.